src/output.rs

```diff
@@ -85,10 +85,10 @@ impl OutputType {
 
         match pagerflags.split_first() {
             Some((pager_name, args)) => {
-                let mut pager_path = PathBuf::from(pager_name);
+                let pager_path = PathBuf::from(pager_name);
 
                 if pager_path.file_stem() == Some(&OsString::from("bat")) {
-                    pager_path = PathBuf::from("less");
+                    panic!("You must not user bat as a pager for bat");
```

Comment:
typo `user` -> `use`

Reply:
You typically want to avoid using `panic` for error handling. This function returns a `Result<…>`, so we should be able to return a proper error here (i.e. `Err(…)`) that would be reported using `bat`s error handling.

You might want to take a look at https://docs.rs/error-chain/0.12.4/error_chain/ which is used for error handling in `bast`

Reply:
Also, we should aim to provide an explanation to the user.

Reply:
I made some changes the new message goes like this:
```
$ bat --pager="bat --style numbers" myfile.txt
[bat error]: the use of bat as a pager is disallow in order to avoid infinite recursion problems
```